<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Choose figure</string>
    </property>
    <widget class="QRadioButton" name="diamondButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Diamond</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="circleButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Circle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="squareButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Square</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="starButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Star</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Choose color</string>
    </property>
    <widget class="QRadioButton" name="redButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="blueButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="greenButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Green</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="isDraw">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
